angular/src/lib/date-picker/date-picker.component.ts

```diff
@@ -183,7 +183,8 @@ export class DatePickerComponent implements OnInit {
   public onPress = (e) => {
     const key = e.keyCode,
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode

Lets try moving away from keyCode as it looks to be deprecated. An alternative would be event.code which would then listen to the word instead of key; "SPACE", "ENTER", etc. @bfbiggs

Reply:
Deprecated, but still required for browsers like IE11. Can we not look for [both](https://github.com/momentum-design/momentum-ui/pull/242/commits/e4fb78d09bc59457ad66bc86c0391c779b7ed117)? Angular cdk is still making an intentional decision to support keyCode until they drop full support for IE.

Reply:
So shall we still use keycode until we give up support IE10 and below? Or do we need to warp keycode and code in a light library? @pauljeter

Reply:
Let's support what the CDK supports and look for both.